harness/propertyHelper.js

```diff
@@ -45,6 +45,19 @@ function verifyProperty(obj, name, desc, options) {
     return true;
   }
 
+  var names = Object.getOwnPropertyNames(desc);
+  for (var i = 0; i < names.length; i++) {
+    assert(
+      names[i] === "value" ||
+        names[i] === "writable" ||
+        names[i] === "enumerable" ||
+        names[i] === "configurable" ||
+        names[i] === "get" ||
+        names[i] === "set",
```

Comment:
No yet. (I have some WIP patches to add accessor property support to `verifyProperty`, though.)

We still need to handle get/set already, because some tests are first passing an accessor property descriptor object to `Object.defineProperty` and then use the same descriptor object with `verifyProperty`.